test/container.test.ts

```diff
@@ -275,17 +288,19 @@ describe('A container', () => {
 
         const promise = tracker.track(payload);
 
+        await expect(new Promise(resolve => { setTimeout(resolve, 5); })).resolves.toBeUndefined();
```

Comment:
Why?

Reply:
I've added a comment to clear things up. Thanks for pointing that out. Let me know if it's clear now!

Reply:
Yes, thanks for the explanation